libraries/AP_HAL_ChibiOS/hwdef/MicoAir743v2/defaults.parm

```diff
@@ -0,0 +1,18 @@
+#BATTERY MONITOR ON
+BATT_MONITOR 4
+
+#enable second OSD type
+OSD_TYPE2 5
+
+#set default PWM type as DSHOT300
```

Comment:
remove...you do not know what ESC capability will be attached

Reply:
Thanks! Fixed!